src/main/java/model/item/Weapon.java

```diff
@@ -1,11 +1,42 @@
 package model.item;
 
+import javafx.beans.binding.Bindings;
+import javafx.beans.binding.IntegerBinding;
+import javafx.beans.property.IntegerProperty;
+import javafx.beans.property.SimpleIntegerProperty;
+
 public class Weapon extends Equipment {
 
-    private String initiative = "0";
-    private String dice = "";
-    private int damage = 0;
-    private int hit = 0;
+    private String initiative;
```

Comment:
Why is this a string?

Reply:
There are special weapons that halve the init of a player.
(The system can't handle that kind of weapon automaticaly at the moment)

Reply:
okay, again, use the decorator pattern here.
initiative should return a float and based on upgrades (which you can decorate a weapon with), this float changes

Reply:
Opened own feature request